zkevm-circuits/src/test_util.rs

```diff
@@ -84,16 +85,13 @@ pub fn test_circuits_using_witness_block(
     }
 
     // run state circuit test
-    // TODO:
-    //     (1) calculate circuit size(like MEMORY_ROWS_MAX etc) from block
-    // rather than hard code  (2) use randomness as one of the circuit
-    // public input, since randomness in state circuit and evm
-    // circuit must be same
+    // TODO: use randomness as one of the circuit public input, since randomness in
+    // state circuit and evm circuit must be same
     if config.enable_state_circuit_test {
-        let state_circuit =
-            StateCircuit::<Fr, true, 2000, 200, 1023, 2000>::new(block.randomness, &block.rws);
-        let prover = MockProver::<Fr>::run(12, &state_circuit, vec![]).unwrap();
-        prover.verify()?;
+        let state_circuit = StateCircuit::new(block.randomness, block.rws);
+        let power_of_randomness = state_circuit.instance();
+        let prover = MockProver::<Fr>::run(18, &state_circuit, power_of_randomness).unwrap();
```

Comment:
For evm circuit, we dynamically decide the k. So many small tests will use only k = 12 (without bitwise fixed table).
If we use k = 18 for every state circuit here(since some big fixed table is needed), the CI time will be quite long.

@han0110 @ed255 Do you have any thoughts in this?

Reply:
what do you think about setting `enable_state_circuit_test` to be false for evm circuit tests, once we add more state circuit tests?

Reply:
> what do you think about setting `enable_state_circuit_test` to be false for evm circuit tests, once we add more state circuit tests?

This sounds like a good tradeoff to me.